sunkit_instruments/goes_xrs/goes_xrs.py

```diff
@@ -72,36 +10,13 @@
     "B": u.Quantity(1e-7, "W/m^2"),
     "A": u.Quantity(1e-8, "W/m^2"),
 }
+
 __all__ = [
     "get_goes_event_list",
-    "calculate_temperature_em",
-    "calculate_radiative_loss_rate",
-    "calculate_xray_luminosity",
```

Comment:
How do you feel about leaving these function in, but have them raise a `NotImplementedError` explaining why it no longer works and asking people to comment on a specific issue if they want this functionality restored?